services/app-web/src/components/SubmissionSummarySection/SubmissionTypeSummarySection/SubmissionTypeSummarySection.tsx

```diff
@@ -24,24 +26,26 @@ export const SubmissionTypeSummarySection = ({
     navigateTo,
     headerChildComponent,
     intiallySubmittedAt,
+    submissionName,
 }: SubmissionTypeSummarySectionProps): React.ReactElement => {
     const isPreviousSubmission = usePreviousSubmission()
     const programNames = statePrograms
         .filter((p) => submission.programIDs.includes(p.id))
         .map((p) => p.name)
+    const isSubmitted = submission.status === 'SUBMITTED'
```

Comment:
I see this `isSubmitted` being defined  in a couple files - can we just throw in a helper?